src/expr-test-util/tests/testdata/rel

```diff
@@ -104,23 +104,24 @@ cat
 ----
 ok
 
-build
-(join
-    [(get y) (get y)]
-    [[#0 #3]]
-    (delta_query [[[0 [#0] (false 1 true 1 (false false false 0 false) 0)] [1 [#0] (false 1 true 1 (false false false 0 false) 0)]]
-                  [[1 [#0] (false 1 true 1 (false false false 0 false) 0)] [0 [#0] (false 1 true 1 (false false false 0 false) 0)]]]))
-----
-----
-Join on=(#0 = #3) type=delta
-  implementation
-    %0:u1 » %0:u1[#0]KA|1| » %1:u1[#0]KA|1|
-    %1:u1 » %1:u1[#0]KA|1| » %0:u1[#0]KA|1|
-  Get u1
-  Get u1
-
-----
-----
+# Commented out when JoinInputCharacteristics got versioned.
```

Comment:
We should form a plan about how we're going to roll out and then uncomment this once we're on a single version.

Reply:
Tbh, nowadays I'm getting quite lazy with these `mzreflect`-based tests, because we mostly decided that we'll just remove this whole test framework, and probably work with the `spec` test framework instead. See https://github.com/MaterializeInc/database-issues/issues/8774